### DISCUSSION TITLE
Add file library (pdf, xls) integration for WYSIWYG interface

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#16869] [Feature Requests]
Hey folks,
Right now we can add images into the WYSIWYG interface.
Can't we add pdf in the same way ?
(with an icon for example, or nothing special).

**Comment by rijkvanzanten** [comment]
The current wysiwyg only supports images and videos

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
